src/pages/Home/Home.js

```diff
@@ -0,0 +1,55 @@
+import './Home.css'
+import React from "react";
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {Link} from "react-router-dom";
+import Header from "../../components/header/Header";
+import logoReddit from "../../assets/logo-reddit.png"
+
+
+function Home() {
+const [posts, setPosts] = useState(null)
```

Comment:
Ik zou een lege array als default value pakken, zodat je de check op regel 35 kwijt bent
```suggestion
const [posts, setPosts] = useState([])
```